src/components/BuildInstitutions.vue

```diff
@@ -113,7 +114,10 @@ export default Vue.extend({
       return this.tempInstitutionSorted[this.previousIndex]
     },
     updateTempInstitutionsFromStore() {
-      this.tempInstitutionSorted = JSON.parse(JSON.stringify(this.institutions))
+      this.tempInstitutionSorted = JSON.parse(JSON.stringify(this.institutions)).map((institution:Institution) => {
+        institution.existing = true;
```

Comment:
institution.existing is not needed. All institutions should be deletable

Reply:
updated